posthog/hogql_queries/test/test_query_runner.py

```diff
@@ -86,7 +87,20 @@ def test_cache_key(self):
         runner = TestQueryRunner(query={"some_attr": "bla"}, team=team)  # type: ignore
 
         cache_key = runner._cache_key()
-        self.assertEqual(cache_key, "cache_f0f2ce8b1f3d107b9671a178b25be2aa")
+        self.assertEqual(cache_key, "cache_33c9ea3098895d5a363a75feefafef06")
+
+    def test_cache_key_runner_subclass(self):
+        TestQueryRunner = self.setup_test_query_runner_class()
+
+        class TestSubclassQueryRunner(TestQueryRunner):  # type: ignore
+            pass
+
+        team = Team.objects.create(pk=42, organization=self.organization)
```

Comment:
I don't know when and where we're running these tests. `pk=42` might already exist. Why not leave it empty and get an autoassigned one?

Reply:
I've added a comment. The first one of these tests already existed, I'd just copied it and adapted it for a slightly different case.

Reply:
Fair. I guess the ID is hardcoded to have a stable cache key. So 🤷